### PR TITLE
Added composer.json file.

### PR DESCRIPTION
**Pull request by rodrigoaguilera**
### Ticket
- https://app.asana.com/0/30975952354191/999106472504087

### Description
- We need a composer file in the repos we want to use as git submodules

### CODE REVIEW

**Review by LucaPipolo — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        {
+            "name": "netzstrategen",
+            "email": "hallo@netzstrategen.com",
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "dist": {
+        "url": "git@github.com:netzstrategen/woocommerce-image-disclaimer.git",
+        "type": "git"
+    },
```
Comment (sun) [inline composer.json]: dist must not be specified statically.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "netzstrategen/image-disclaimer",
```
Comment (sun) [inline composer.json]: Only the wordpress prefix is removed.
```suggestion
    "name": "netzstrategen/woocommerce-image-disclaimer",
```

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "dist": {
+        "url": "git@github.com:netzstrategen/woocommerce-image-disclaimer.git",
+        "type": "git"
+    },
+    "require": {
+        "php": ">=7.0.0",
+        "composer/installers": "^1"
```
Comment (sun) [inline composer.json]: Please remove; this is a requirement of the root project that wants to use this plugin, but not of this plugin.

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            "email": "hallo@netzstrategen.com",
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "dist": {
+        "url": "git@github.com:netzstrategen/woocommerce-image-disclaimer.git",
+        "type": "git"
+    },
+    "require": {
+        "php": ">=7.0.0",
```
Comment (sun) [inline composer.json]: We actually require at least 7.1

**Review by sun — CHANGES_REQUESTED** [review]

**Comment by LucaPipolo** [comment]
@sun — I addressed your requests. Can you please have a double check to it and approve if happy? Thanks.

File: composer.json (hunk 1 above)
Comment (sun) [reply composer.json]: dist is still contained.

**Review by sun — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (LucaPipolo) [reply composer.json]: Ah sorry, I wrongly understood you mean to remove only the `git` type. I fixed this into a new fixup commit.

**Review by sun — APPROVED** [review]

**Pull request merged**